packages/sdk-components-react-radix/src/collapsible.tsx

```diff
@@ -35,3 +40,70 @@ export const CollapsibleTrigger = forwardRef<
 export const CollapsibleContent: ForwardRefExoticComponent<
   Omit<ComponentPropsWithRef<typeof Content>, "asChild">
 > = Content;
+
+/* BUILDER HOOKS */
+
+const namespace = "@webstudio-is/sdk-components-react-radix";
+
+const onCollapsibleContentMatched = (
```

Comment:
is this something we can generalize? having sdk author to write this is nearly impossible task

Reply:
name is a little bit confusing, generally it's near same as html `closest`

Reply:
Probably deoptimizing it and splitting into iterate over selector and `closest()` method would look more readable

Reply:
I think we need to do something about it right away. Because we need to open up the api soon and we can't be scaring people with such logic that they need to write